src/script/campaign/mod/vco-listeners.lua

```diff
@@ -2,8 +2,27 @@ local vco = core:get_static_object("vco");
 
 -- COMMON --
 
-local function earned_x_pooled_resource(resource, amount)
-  return true
+local function faction_earned_x_pooled_resource(resource, amount)
+  local amount_of_resource_earned = cm:get_saved_value("vco_pooled_resource_earned" .. resource) or 0;
+  local amount_of_resource_earned_updated = amount_of_resource_earned;
```

Comment:
No need to declate it here as it is local to the if. You are also overriding the value anyway.